ci-operator/step-registry/ipi/conf/aws/oidc-creds/ipi-conf-aws-oidc-creds-commands.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+PREFIX="${SHARED_DIR}/manifest"
+cat >> "${PREFIX}_cluster-authentication-02-config.yaml" << EOF
+apiVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+  name: cluster
+spec:
+  serviceAccountIssuer: https://<something>-oidc-discovery.s3.amazonaws.com
+
+EOF
+
+CONFIG="[default]
+role_arn = arn:aws:iam::<something>:role/<something>-cred-minter-s3-role
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
```

Comment:
probably should define this token path up at the top of the bash script since it is used so many times below